core/server/src/main/java/alluxio/worker/netty/DataServerReadHandler.java

```diff
@@ -79,6 +79,7 @@
   /**
    * This is only updated in the channel event loop thread when a read request starts or cancelled.
    * No need to be locked.
+   * When it is accessed outside of the event loop thread, make sure it is not null.
```

Comment:
I've updated the comment. In our codebase, all the netty channel handlers are the only functions executed in the event loop thread. On the contrary, the packet reader in this class is not executed in the event loop thread.